### PR TITLE
chore(cluster) code clean

### PR DESCRIPTION
**Pull request by chronolaw**
### Summary

Just some code/style clean.

### CODE REVIEW

**Review by Suika-Kong — APPROVED** [review]

File: kong/tools/grpc.lua [hunk 1]
```diff
@@ -11,6 +11,7 @@ local type = type
 local pcall = pcall
 local error = error
 local tostring = tostring
+local ipairs = ipairs
```
[the remark this answers is not in the document]
Comment (flrgh) [reply kong/tools/grpc.lua:14]: Interesting, TIL!

[the remark this answers is not in the document]
Comment (chronolaw) [reply kong/tools/grpc.lua:14]: Thanks, I learned. but it is a harmless idiom, right?

[the remark this answers is not in the document]
Comment (javierguerragiraldez) [reply kong/tools/grpc.lua:14]: > harmless idiom, right?

absolutely!

[the remark this answers is not in the document]
Comment (chronolaw) [reply kong/tools/grpc.lua:14]: Thank your Javier, you are so knowledgeable.

**Review by flrgh — APPROVED** [review]

**Review by kikito — APPROVED** [review]

**Pull request merged**